lib/bloc/user_profile/user_profile_bloc.dart

```diff
@@ -39,6 +46,10 @@ class UserProfileBloc {
   final _fiatConversionController = new BehaviorSubject<String>();
   Sink<String> get fiatConversionSink => _fiatConversionController.sink;
 
+  final _securityModelController = new BehaviorSubject<SecurityModel>();
+  Sink<SecurityModel> get securityModelSink => _securityModelController.sink;
```

Comment:
We don't need this controller and sink. We should use the `_userController` and push the changes user there.